dogfood/main.tf

```diff
@@ -213,6 +216,16 @@ resource "coder_app" "code-server" {
   }
 }
 
+resource "coder_app" "filebrowser" {
+  agent_id     = coder_agent.dev.id
+  display_name = "File Browser"
+  slug         = "filebrowser"
+  url          = "http://localhost:8080/"
+  icon         = "https://raw.githubusercontent.com/matifali/logos/main/database.svg"
+  subdomain    = true
```

Comment:
I was not able to use it without `subdomain=true`

Reply:
It may be possible, following [this](https://github.com/filebrowser/filebrowser/issues/2477#issuecomment-1637038616) approach and I was able to start it on a subpath with a minor issue. (somehow coder repeats app path 2 times)